net/openssh/patches/010-better_fzero-call-detection.patch

```diff
@@ -0,0 +1,30 @@
+--- a/m4/openssh.m4
++++ b/m4/openssh.m4
```

Comment:
We can not accept patches without Git header. We dont know if it is backport or not.

Could you please use it like this:
https://github.com/openssh/openssh-portable/commit/1036d77b34a5fa15e56f516b81b9928006848cbd.patch

and then refresh it, please?